web/blueprints/user/__init__.py

```diff
@@ -653,7 +653,7 @@ def reset_password(user_id):
     form = UserResetPasswordForm()
     myUser = User.q.get(user_id)
     if form.validate_on_submit():
-        plain_password = lib.user.reset_password(myUser)
+        plain_password = lib.user.reset_password(myUser, current_user)
```

Comment:
Please be more descriptive and add a keyword argument (`processor=current_user`). At first sight you just pass two users whose role as arguments is not easily distinguishable.

Reply:
good point.